.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: ruby
```

Comment:
Adding `sudo: false` enables the new container based travis ci service.

http://docs.travis-ci.com/user/workers/container-based-infrastructure/